src/vm/dllimportcallback.cpp

```diff
@@ -1416,6 +1416,7 @@ VOID UMThunkMarshInfo::RunTimeInit()
     {
         // caller pop
         m_cbRetPop = 0;
+        m_cbActualArgSize += numRegistersUsed * STACK_ELEM_SIZE;
```

Comment:
I've tried both ways, but I get the same results. Here is a print of pStubMD and pMD:
```
(gdb) p *pStubMD
$1 = {m_pszDebugMethodName = 0xf7929a45 "IL_STUB_ReversePInvoke", m_pszDebugClassName = 0xf79299fa "DomainBoundILStubClass", m_pszDebugMethodSignature = 0xf244c22c "instance bool *(int32&)", 
  m_pDebugMethodTable = {m_addr = 4064572276}, m_GcCover = 0x0, m_wFlags3AndTokenRemainder = 0, m_chunkIndex = 0 '\000', m_bFlags2 = 8 '\b', m_wSlotNumber = 65535, m_wFlags = 32815, 
  static s_ClassificationSizeTable = 0xf78b4ee0 <MethodDesc::s_ClassificationSizeTable>}
(gdb) p *pMD
$2 = {m_pszDebugMethodName = 0xf4e58da8 "Invoke", m_pszDebugClassName = 0xf62904bc "Cdeclcaller", m_pszDebugMethodSignature = 0xf4e58e2c "instance bool *(int32&)", m_pDebugMethodTable = {m_addr = 4129883552}, 
  m_GcCover = 0x0, m_wFlags3AndTokenRemainder = 16441, m_chunkIndex = 9 '\t', m_bFlags2 = 0 '\000', m_wSlotNumber = 12, m_wFlags = 3, 
  static s_ClassificationSizeTable = 0xf78b4ee0 <MethodDesc::s_ClassificationSizeTable>}

```